Spanner/tests/Perf/Operation.php

```diff
@@ -0,0 +1,267 @@
+<?php
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Spanner\Tests\Perf;
+
+use Google\Cloud\Spanner\Database;
+
+class Operation
+{
+    /**
+     * @var string
+     */
+    private $database;
+
+    /**
+     * @var array
+     */
+    private $parameters;
+
+    /**
+     * float
+     */
+    private $totalWeight;
+
+    /**
+     * @var array
+     */
+    private $weights;
+
+    /**
+     * @var array
+     */
+    private $operations;
+
+    /**
+     * @var array
+     */
+    private $latency;
+
+    /**
+     * @var array
+     */
+    private $keys;
+
+    /**
+     * @param string $database
+     * @param array $parameters
+     * @param float $totalWeight
+     * @param array $weights
+     * @param array $operations
+     * @param array $latency
+     */
+    public function __construct(
+        $database,
+        array $parameters,
+        $totalWeight,
+        array $weights,
+        array $operations,
+        array $latency
+    ) {
+        $this->database = $database;
+        $this->parameters = $parameters;
+        $this->totalWeight = $totalWeight;
+        $this->weights = $weights;
+        $this->operations = $operations;
+        $this->latency = $latency;
+    }
+
+    public function load()
+    {
+        $this->keys = [];
+
+        $startTime = microtime(true);
+        $snapshot = $this->database->snapshot();
+
+        // Kind of assuming that id is always name of PK in whatever table you choose
+        $results = $snapshot->execute('SELECT id FROM ' . $this->parameters['table']);
+        foreach ($results as $row) {
+            $this->keys[] = $row['id'];
+        }
+
+        return microtime(true) - $startTime;
```

Comment:
Why are we calculating this and not using it anywhere?

Reply:
See other comment. Without an intimate knowledge of the tool, I was reticent to remove functionality the potential usefulness of which I could not easily assess.

Reply:
It isn't being used though, right? Why keep it then, or am I missing something?